packages/worker/src/api/index.js

```diff
@@ -106,7 +106,10 @@ router
     if (ctx.publicEndpoint) {
       return next()
     }
-    if ((!ctx.isAuthenticated || !ctx.user.budibaseAccess) && !ctx.internal) {
+    if (
+      (!ctx.isAuthenticated || (ctx.user && !ctx.user.budibaseAccess)) &&
```

Comment:
The removal of the brackets around `(!ctx.isAuthenticated || !ctx.user.budibaseAccess)` is this desired? Is it not `(!ctx.isAuthenticated || (ctx.user && !ctx.user.budibaseAccess)) && ...`?

Reply:
ignore!